src/rest-server/src/controllers/vc.js

```diff
@@ -23,7 +23,9 @@ const createError = require('../util/error');
  * Validation, not allow operation to "default" vc.
  */
 const validate = (req, res, next, vcName) => {
-  if (vcName === 'default' && req.method !== 'GET') {
+  if (! /^[A-Za-z0-9_,]+$/.test(vcName)) {
```

Comment:
Can `vcName` contains comma `,`?
```suggestion
  if (! /^[A-Za-z0-9_]+$/.test(vcName)) {
```

Reply:
Fixed